src/commands/coverage/upload.ts

```diff
@@ -38,14 +37,35 @@ import {
   renderSuccessfulUploadCommand,
   renderUpload,
 } from './renderer'
-import {detectFormat, isFile, validateCoverageReport} from './utils'
+import {detectFormat, validateCoverageReport} from './utils'
 
 const TRACE_ID_HTTP_HEADER = 'x-datadog-trace-id'
 const PARENT_ID_HTTP_HEADER = 'x-datadog-parent-id'
 const errorCodesStopUpload = [400, 403]
 
 const MAX_REPORTS_PER_REQUEST = 10
 
+const IS_COVERAGE_REPORT = (file: string): boolean => {
```

Comment:
nitpick: it's a bit surprising that a function is in upper case and camel case. Could we do `function isCoverageReport`?

Reply:
renamed